.github/workflows/scheduled.yaml

```diff
@@ -22,3 +23,14 @@ jobs:
 
       - name: Run
         run: cargo test --test wasm-generation --test standard
+
+      - name: Add link to the failure issue
+        uses: jayqi/failed-build-issue-action@v1
+        if: ${{ failure() }}
+        with:
+          github-token: ${{ secrets.GH_TOKEN }}
+          label-name: "property-testing-failure"
+          title-template: "Property testing coverage has failed"
+          body-template: |
+            FYI & A: @stacks-network/clarity-wasm
```

Comment:
I don't think this line is necessary.

Reply:
No need to @ mention the team. Everyone can set notifications for new issues on their own if they want.

Reply:
Ok. Makes sense. 👍 Changed.